### PR TITLE
MWPW-138565 | Gnav overlapping with popup promobar

### PR DESCRIPTION
**Pull request by aishwaryamathuria**
Popup promobar takes a 16px gap from the bottom to have a floating effect. The current bottom attribute is making the promobar overlap with the Gnav.
Changing the bottom attribute and applying it on section so that the promobar does not overlap with gnav when previewed in fragment.

Sample page links -
https://mwpw-138565--milo--aishwaryamathuria.hlx.page/drafts/mathuria/promobar/creativecloud
https://mwpw-138565--milo--aishwaryamathuria.hlx.page/drafts/mathuria/promobar/creativecloud-default

Resolves: [MWPW-138565](https://jira.corp.adobe.com/browse/MWPW-138565)

**Test URLs:**
- Before: https://main--milo--aishwaryamathuria.hlx.page/drafts/mathuria/promobar/fragments/popup-promo-aside?martech=off
- After: https://mwpw-138565--milo--aishwaryamathuria.hlx.page/drafts/mathuria/promobar/fragments/popup-promo-aside?martech=off

### CODE REVIEW

**Comment by asonnalagi** [comment]
Verified the issue with @aishwaryamathuria Domain, Its fine now.
https://mwpw-138565--milo--aishwaryamathuria.hlx.page/drafts/Anupama/promo-close/promo-with-footer-off
image

File: libs/blocks/section-metadata/section-metadata.css [hunk 1]
```diff
@@ -172,6 +172,10 @@
   z-index: 4;
 }
 
+.section.sticky-bottom.popup-sticky {
```
[the remark this answers is not in the document]
Comment (aishwaryamathuria) [reply libs/blocks/section-metadata/section-metadata.css]: @overmyheadandbody I have fixed the after link in the description. 
I have made the suggested change and have changed .popup-sticky to .popup-sticky-section

[the remark this answers is not in the document]
Comment (skumar09) [reply libs/blocks/section-metadata/section-metadata.css]: @aishwaryamathuria : 
`Merch card` block tests scripts are failing on your branch, I am observing` price` on the cards are not getting displayed can you please review... 

image

image

Info: Nala run job is showing failures in the run log, but job run is showing pass ( currently I am debugging on this)

cc : @rgclayton @auniverseaway

[the remark this answers is not in the document]
Comment (aishwaryamathuria) [reply libs/blocks/section-metadata/section-metadata.css]: @skumar09 It seems to be throwing log error for other PRs as well. Could you confirm if there is some issue due to which errors are thrown by the Nala automation?

**Review by overmyheadandbody — COMMENTED** [review]
Is there a test page using this fragment, to ensure previous functionality remains intact?

File: libs/blocks/section-metadata/section-metadata.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (skumar09) [reply libs/blocks/section-metadata/section-metadata.css]: > @skumar09 It seems to be throwing log error for other PRs as well. Could you confirm if there is some issue due to which errors are thrown by the Nala automation?

@aishwaryamathuria : 
- Merch-cards tests are failing due to CORS issue on all PR branches ( except adobecom), 
- I have commented out the price validation lines as a short-term workaround ( i.e. till the CORS issue is resolved)

**Comment by aishwaryamathuria** [comment]
> Is there a test page using this fragment, to ensure previous functionality remains intact?

@overmyheadandbody Following is a test page using this functionality that can be used to verify the existing behaviour -
https://main--milo--aishwaryamathuria.hlx.page/drafts/mathuria/promobar/creativecloud

**Review by overmyheadandbody — APPROVED** [review]

**Review by ryanmparrish — APPROVED** [review]

File: libs/blocks/section-metadata/section-metadata.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (rgclayton) [reply libs/blocks/section-metadata/section-metadata.css]: > I see a previous example with `.section.sticky-bottom.promo-sticky-section`, so maybe we can update `.popup-sticky` to `.popup-sticky-section`.

Nit: To me this reads overly verbose and makes me think twice about what is going on. We know it's a section already from the start of that line `.section`. Adding another class name `-section` seems unnecessary. I think the original suggestion made more sense to me. 

To me something like `.sticky-bottom.popup` or `.section.sticky-bottom.popup` creates a more simple pattern moving forward.

`.sticky-bottom.popup`
`.sticky-top.popup`
`.sticky-left.popup`
`.sticky-right.popup`

cc: @overmyheadandbody @ryanmparrish

[the remark this answers is not in the document]
Comment (overmyheadandbody) [reply libs/blocks/section-metadata/section-metadata.css]: No objections from my side, as long as we stick to a pattern. There was a similar class name used in those styles, so my comment was just around alignment. So I'd just like to have a similar naming schema for both to reduce confusion.

**Review by nishantka — APPROVED** [review]

**Review by sukamat — APPROVED** [review]

**Comment by aishwaryamathuria** [comment]
@Sartxi Could you share your review on the PR ?

File: libs/blocks/section-metadata/section-metadata.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (aishwaryamathuria) [reply libs/blocks/section-metadata/section-metadata.css]: @rgclayton I have made the suggested change and updated the class name from verbose promo-sticky-section to popup. Could you review the same?

**Review by rgclayton — APPROVED** [review]

**Pull request merged**